config/locales/en/controllers/account_links.yml

```diff
@@ -0,0 +1,15 @@
+---
+en:
+  account_links:
+    controller:
+      add_shared_user:
+        granted_push: Granted push privileges to %{user}.
+        share_duplicate: This account link is already shared with %{user}.
+      create:
+        success_notice: Account link was successfully created.
```

Comment:
Those flash messages are quite common. Therefore, one could think of extracting those to a common message and just passing the human model name (compare [the existing mechanism in CodeOcean](https://github.com/openHPI/codeocean/blob/8fc5123bae62f2c2901e9230cd693ef77bc1222b/config/locales/en.yml#L943-L946)). This could further reduce the number of translations (and thereby ensure they are consistent).

Reply:
Thats true. I replaced those translations.